### PR TITLE
Issue#1060 Added sample snippet for TableAdmin::CheckConsistency

### PR DESCRIPTION
**Pull request by roopak-qlogic**
Added sample snippet for TableAdmin::CheckConsistency

Fixes issue #1060 

---
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-cpp/1097)

### CODE REVIEW

File: google/cloud/bigtable/examples/bigtable_samples.cc [hunk 1]
```diff
@@ -302,6 +302,22 @@ void WaitForConsistencyCheck(google::cloud::bigtable::TableAdmin admin,
...
+void CheckConsistency(google::cloud::bigtable::TableAdmin admin,
+                      std::string const& table_id_param) {
+  google::cloud::bigtable::TableId table_id(table_id_param);
+  google::cloud::bigtable::ConsistencyToken consistency_token(
+      admin.GenerateConsistencyToken(table_id.get()));
+  auto result = admin.CheckConsistency(table_id, consistency_token);
+  if (result) {
+    std::cout << "Table is consistent" << std::endl;
+  } else {
+    std::cout << "Table is not consistent" << std::endl;
```
Comment (coryan) [inline google/cloud/bigtable/examples/bigtable_samples.cc]: Please change this example to receive the consistency token as a parameter (it is basically a string).  And change the output here to "Table is not yet consistent, try again later with the same token."

**Comment by roopak-qlogic** [comment]
1) I do not know what default value to be passed as consistency_token, so used value "test-token", Which is causing an exception thrown by the function, and do not know How to handle the exception raised by admin.CheckConsistency() function.
2) Also Is it ok to pass the token from main function, instead of getting value in main function as a parameter. If I change it to pass thru runtime_example_utils.sh then multiple changes need to be done for handling Parameters.

**Comment by coryan** [comment]
> I do not know what default value to be passed as consistency_token, so used value "test-token", Which is causing an exception thrown by the function, and do not know How to handle the exception raised by admin.CheckConsistency() function.

What we should do is have one example that prints out the consistency token obtained with `GenerateConsistencyToken`, capture the output from that example, and use that as the input for the example you are trying to write.

> Also Is it ok to pass the token from main function, instead of getting value in main function as a parameter.

I do not think that would work.  If we want to tell the user "Please try again with the same token." the user must be able to pass in the token.

It might be better to break this PR into a few (see #1107):

- In a separate PR, let's split the TableAdmin examples out of  `bigtable_samples.cc`.
- Then change that new samples file to use `argc` and `argv` in each sample function.
- Then make this change.

File: google/cloud/bigtable/examples/bigtable_samples.cc [hunk 2]
```diff
@@ -206,4 +206,4 @@ int main(int argc, char* argv[]) try {
   std::cerr << "Standard C++ exception raised: " << ex.what() << std::endl;
   return 1;
 }
-//! [all code]
+//! [all code]
```
Comment (coryan) [inline google/cloud/bigtable/examples/bigtable_samples.cc]: missing newline at end of file.

File: google/cloud/bigtable/examples/run_examples_utils.sh [hunk 3]
```diff
@@ -360,4 +360,4 @@ run_hello_app_profile_example() {
   # Verify that calling without a command produces the right exit status and
   # some kind of Usage message.
   run_example_usage ./bigtable_hello_app_profile
-}
+}
```
Comment (coryan) [inline google/cloud/bigtable/examples/run_examples_utils.sh]: missing newline at end of file.

File: google/cloud/bigtable/table_admin.h [hunk 4]
```diff
@@ -395,4 +395,4 @@ class TableAdmin {
 }  // namespace cloud
 }  // namespace google
 
-#endif  // GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGTABLE_TABLE_ADMIN_H_
+#endif  // GOOGLE_CLOUD_CPP_GOOGLE_CLOUD_BIGTABLE_TABLE_ADMIN_H_
```
Comment (coryan) [inline google/cloud/bigtable/table_admin.h]: missing newline at end of file.

**Comment by roopak-qlogic** [comment]
here I did just solved the conflict, did not do any changes, But we need to fix one review comment here, which does not included in any PR, that is,   "What we should do is have one example that prints out the consistency token obtained with GenerateConsistencyToken, capture the output from that example, and use that as the input for the example you are trying to write.".
Here waiting for below PR should merge (snippets for GenerateConsistencyToken)

https://github.com/GoogleCloudPlatform/google-cloud-cpp/pull/1173

File: google/cloud/bigtable/examples/table_admin_snippets.cc [hunk 5]
```diff
@@ -215,20 +215,19 @@ void WaitForConsistencyCheck(google::cloud::bigtable::TableAdmin admin,
...
                       char* argv[]) {
-  if (argc != 3) {
-    throw Usage{
-        "check-consistency: <project-id> <instance-id> <table-id> "
-        "<consistency_token>"};
+  if (argc != 2) {
+    throw Usage{"check-consistency: <project-id> <instance-id> <table-id>"};
   }
   std::string const table_id_param = ConsumeArg(argc, argv);
-  std::string const consistency_token_param = ConsumeArg(argc, argv);
```
Comment (coryan) [inline google/cloud/bigtable/examples/table_admin_snippets.cc]: Why did we drop this?

Comment (roopak-qlogic) [reply google/cloud/bigtable/examples/table_admin_snippets.cc]: My thought was as below 
 
1. if we do no want to add in ci build, then we can keep this user token as a function parameter
2. if we want to add in ci build, then seems no use of user token as a function parameter, and use GenerateConsistencyToken inside the example only. 
3. if we want to add in ci build, and also keep user token as a function parameter, then need to check more, how to do that, or if you have any suggestion.

Comment (coryan) [reply google/cloud/bigtable/examples/table_admin_snippets.cc]: I think we can do option 3 without much effort:

https://github.com/GoogleCloudPlatform/google-cloud-cpp/blob/ad0601fd8e3b8cee786fb5bb472f4479322740a3/google/cloud/storage/examples/run_examples_utils.sh#L231

**Pull request merged**